foundry.toml

```diff
@@ -11,13 +11,15 @@ evm_version = "paris"                                       # set the EVM target
 optimizer = true                                            # enable the solc optimiser
 optimizer_runs = 10_000_000                                 # the number of optimiser runs
 via_ir = false                                              # disable compilation pipeline to go through the Yul intermediate representation
+bytecode_hash = "none"                                      # remove the metadata hash from the bytecode
+fuzz = { runs = 256 }                                       # the number of fuzz runs for tests
+invariant = { runs = 256, depth = 15 }                      # the number of runs and calls (executed in one run) for each invariant test group
 verbosity = 3                                               # the verbosity of tests
 fs_permissions = [{ access = "read-write", path = "./" }]   # set read-write access to project root
-bytecode_hash = "none"                                      # remove the metadata hash from the bytecode
 
 ## default overrides for the CI runs
 [profile.ci]
 force = true                                                # always perform a clean build
 fuzz = { runs = 10_000 }                                    # increase the number of fuzz runs
-invariant = { runs = 5_000 }                                # increase the number of runs that must execute for each invariant test group
+invariant = { runs = 1_000, depth = 250 }                   # increase the number of runs and calls (executed in one run) for each invariant test group
```

Comment:
I would suggest 500/500 here—from some benchmarks the foundry team is working on, depth seems to be a bigger contributor to finding bugs than number of runs. So 500/500 gives the same total of 250k calls, but a better chance at finding bugs.
```suggestion
invariant = { runs = 500, depth = 500 }                   # increase the number of runs (while preserving the default depth) for each invariant test group
```

Reply:
Makes sense! Fixed.